qiskit/circuit/library/standard_gates/i.py

```diff
@@ -20,12 +20,12 @@
 class IGate(Gate):
     r"""Identity gate.
 
-    Identity gate corresponds to a single-qubit gate wait cycle,
-    and should not be optimized or unrolled (it is an opaque gate).
+    Logically this gate is a quantum no-op and the transpiler will largely treat it as such, but
+    some older hardware may choose to interpret it as a short-hand for the shortest-possible delay
+    cycle.
```

Comment:
```suggestion
    This gate is a quantum no-op and represents an identity channel on a single qubit.
    It will not have a logical effect on the qubit state and will likely be optimized away by the compiler at all optimization levels greater than zero.
    Hardware backends may implement this instruction with a potentially non-zero idle duration, for example a delay of length equal to that of a single-qubit gate, but this behavior will depend on the selected backend.
```

Reply:
Done (give or take) in d6e385457.